tools/cli/wskaction.py

```diff
@@ -170,7 +174,10 @@ def getExec(self, args, props):
             exe = self.getActionExec(args, props, pipeAction)
         elif args.artifact is not None and os.path.isfile(args.artifact):
             contents = open(args.artifact, 'rb').read()
-            if args.artifact.endswith('.swift'):
+            if args.kind in ['swift:3','swift:3.0','swift:3.0.0']:
```

Comment:
I'm not sure what syntax change is being proposed.

Prior to this PR, we supported "kind" = "nodejs", "docker", "java", "swift"

This PR was going to add at least "swift:3" to that list.

I don't want this PR to be a breaking API change, so I don't want to force kind to always be "name:version" , at least for now.

Clear?   Did I misunderstand the question?